rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1799,33 +1799,37 @@ void game_list_frame::BatchRemoveShaderCaches()
 QPixmap game_list_frame::PaintedPixmap(QPixmap icon, bool paint_config_icon, bool paint_pad_config_icon, const QColor& compatibility_color)
 {
 	const qreal device_pixel_ratio = devicePixelRatioF();
-
-	// Let's upscale the original icon to at least fit into the outer rect of the size of PS3's ICON0.PNG
-	if (icon.width() < 320 || icon.height() < 176)
-	{
-		icon = icon.scaled(320, 176, Qt::KeepAspectRatio);
-	}
-
-	QSize canvas_size(icon.size());
+	QSize canvas_size(320, 176);
 	QPoint target_pos;
 
-	// Calculate the centered size and position of the icon on our canvas.
-	if (icon.width() != 320 || icon.height() != 176)
+	if (!icon.isNull())
```

Comment:
Maybe this should just be early exit

Reply:
no. because then we can't paint anything.
Keep in mind that there are 2 icons painted and a background too.